src/components/Products/ProductsListItem.vue

```diff
@@ -16,7 +16,7 @@
       <span class="product-list-item__description__price">${{ price }}</span>
     </div>
     <div class="product-list-item__rating">
-      <img src="../../assets/star.png" /> {{ rate }}
+      <img src="../../assets/star.png" /> {{ rating }}
```

Comment:
it would be nicer to wrap strings in some html elements which would describe their content, or at least a span, like
`<em>{{ comething }}</em>`